tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 
 ax.plot(freqs, psds_mean, color='k')
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
```

Comment:
this example is too long. Keep the complexity low. Also for me it makes sensor to use this robust psd_welch with Raw data and not much for Epochs as it seems like epoching Epochs... this suggest that we should add the combine parameter to raw.plot_psd and simply document this usecase.